source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/with-data.tsx

```diff
@@ -19,7 +19,14 @@ const mapNetworkToProps = (passedProps) => {
   return {
     cacheKey: `project-${id}`,
     project: [q => buildFindRecord(q, PROJECT, id), buildOptions({
-      include: ['products.product-artifacts', 'organization.organization-product-definitions.product-definition', GROUP, 'owner', REVIEWERS, 'type']
+      include: [
+        'products.product-artifacts',
+        'organization.organization-product-definitions.product-definition',
+        GROUP,
```

Comment:
never liked the idea of includes with variables.

GROUP => 'group' ... especially if they are just hiding strings... I suggest we change it to just regular text.

Reply:
variables are better, they are just a little misused at the moment.
the goal is to reduce misspellings, and catch errors at build time, rather than randomly at runtime.
we need a better solution for dot notation though.